pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "p5"
+version = "0.8.4"
+description = "p5 is a Python library that provides high level drawing functionality to help you quickly create simulations and interactive art using Python"
+authors = ["Abhik Pal <theabhikpal@gmail.com>"]
+readme = "README.rst"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+freetype-py = "^2.1.0"
```

Comment:
Why `freetype-py` is added a direct dependency?

Reply:
Removed